app/DoctrineMigrations/Version20200416135127.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20200416135127 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE INDEX idx_vetted_second_factor_type ON vetted_second_factor (type)');
+        $this->addSql('DROP TABLE ra_candidate');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        $this->throwIrreversibleMigrationException('This migration is irreversible and cannot be reverted because it will need a replay on the RACandidateProjector.');
+
+        //$this->addSql('DROP INDEX idx_vetted_second_factor_type ON vetted_second_factor');
```

Comment:
Remove this permanently? If you want to keep it for future reference, then please add a comment.

Reply:
Removed it completely